### ISSUE TITLE
UI components are not completed in webPLP

### ISSUE THREAD

**Issue by tobielf** [Progressive-Learning-Platform/PLPTool6#93]

**Comment by tobielf** [comment]
@tanpuresiddhant Please ask Dr. Sohoni the detail UI components he wants to add to WebUI. If this task is too big, we can separate it into small pieces.

**Comment by tusharpandit18** [comment]
Do you have anything that I can start with?

**Comment by tobielf** [comment]
Start with the requirement engineering first. Analysis the requirement from the client, then you know how many tasks you need to complete.

**Comment by tusharpandit18** [comment]
Ok. Will begin after getting the inputs from Dr. Sohoni

**Comment by tusharpandit18** [comment]
Hello @tanpuresiddhant / @tobielf 
I see a branch, fixing-bug-93, that already exists. Should I use the same branch to continue working on the UI components or create a new one?

**Comment by tanpuresiddhant** [comment]
Use the same and update it once you complete the changes.

**Comment by tusharpandit18** [comment]
Ok. Also, is there anything specific you want me to work on? Like any high priority UI component or feature?

**Comment by tobielf** [comment]
Did you ask Dr. Sohoni for the missing UI components list?

**Comment by tusharpandit18** [comment]
Yes. He has suggested on making the UI responsive first. Have started working on it. 
I made a few changes to the HTML, but there are a few Javascript errors I see in the console. As a result, the changes I make are not reflected on the UI. I am looking into those errors. Also, have updated @tanpuresiddhant about the same. He is looking into those JS errors as well.
